### PR TITLE
feat: add inline schema to ui5*.yaml files to support yaml-language-server

### PR DESCRIPTION
**Pull request by ullasholla**
This PR adds an inline schema to `ui5*.yaml` files to support [`yaml-language-server`](https://github.com/redhat-developer/yaml-language-server#using-inlined-schema). This library is used by IDEs to support features like autocomplete and linting  for YAML files.

The following comment is added to the top of the file for this:
```yaml
# yaml-language-server: $schema=https://sap.github.io/ui5-tooling/schema/ui5.yaml.json

```

The following writers are affected:
* `fiori-freestyle-writer`: ui5-local.yaml
* `odata-service-writer`: ui5-mock.yaml
* `ui5-application-writer`: ui5.yaml, ui5-local.yaml

This PR also updates the affected test snapshots and input files.

### CODE REVIEW

File: packages/fiori-freestyle-writer/templates/common/add/ui5-local.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# yaml-language-server: $schema=https://sap.github.io/ui5-tooling/schema/ui5.yaml.json
+
 specVersion: "2.5"
 metadata:
```
Comment (tobiasqueck) [inline packages/fiori-freestyle-writer/templates/common/add/ui5-local.yaml:4]: I know it is not exactly relevant for this PR but why do we have that file here? This file is defined in th`ui5-application-writer` so why do we redefine it here?

**Review by tobiasqueck — CHANGES_REQUESTED** [review]

File: packages/fiori-freestyle-writer/templates/common/add/ui5-local.yaml (hunk 1 above)
Comment (tobiasqueck) [reply packages/fiori-freestyle-writer/templates/common/add/ui5-local.yaml]: @ullasholla I can answer my own question. The files is obsolete, I have removed it in 48d6fe7  and all snapshot tests run through as before.

**Review by tobiasqueck — APPROVED** [review]

**Review by devinea — APPROVED** [review]

**Comment by devinea** [comment]
FYI @IainSAP @theStiggySpd

File: packages/fiori-freestyle-writer/templates/common/add/ui5-local.yaml (hunk 1 above)
Comment (ullasholla) [reply packages/fiori-freestyle-writer/templates/common/add/ui5-local.yaml]: Thanks. Going to mark this as resolved.

**Pull request merged**